src/database.py

```diff
@@ -0,0 +1,20 @@
+import mysql.connector
+
+testDB = mysql.connector.connect(
+    host = "localhost",
+    user = "root",
+    # ja ich weiss, gibt besser arten mit passwort umzugehen ^^
+    passwd = "",
+    database = "noHateDB",
+    )
+
+cursor = testDB.cursor()
+# cursor.execute("CREATE DATABASE noHateDB")
+# cursor.execute("CREATE TABLE testWords(sID INTEGER AUTO_INCREMENT PRIMARY KEY, command TEXT, date DATETIME, label TEXT)")
+# cursor.execute("DROP TABLE testWords")
+
+def insertRow(command, date, label):
+    insertCommand = "INSERT INTO testWords(command, date, label) VALUES(%s, %s, %s)"
```

Comment:
ändere den table namen bitte in 'comments' und definiere dafür eine feste variable unter der defition vom cursor. das label kannst du hier weglassen.

auch hier wieder unterstrich statt camel case -> insert_command